lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -25,6 +27,16 @@ def check_inflected(node)
           end
         end
 
+        def replaceable_predicate?(predicate)
+          case predicate.method_name.to_s
+          when 'is_a?', 'instance_of?', 'include?',
+            'respond_to?', 'exist?', 'exists?', /^has_/
```

Comment:
Match to multiline is unexpected, right?

```suggestion
            'respond_to?', 'exist?', 'exists?', /\Ahas_/
```

Reply:
Indeed it is. Thank you so much.